vernac/comInductive.mli

```diff
@@ -81,8 +81,8 @@ val template_polymorphism_candidate
    monomorphic universe context that can be made parametric in its
    conclusion sort, if one is given. *)
 
-val maybe_unify_params_in : Environ.env -> Evd.evar_map -> ninds:int -> nparams:int
+val maybe_unify_params_in : Environ.env -> Evd.evar_map -> ninds:int -> nparams:int -> binders:int
```

Comment:
Is this an API that could be used by some external plugins? If yes, I cannot backport this PR as is.

Reply:
Very unlikely IMHO, in fact this function was introduced in https://github.com/coq/coq/pull/11579 , and from a quick look it should disappear if some more refactoring among record and cominductive would happen.

Reply:
Thanks! It sounds reasonably safe to backport then 😃